packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v2.3.0-deploy-contract-from-json-json-object.test.ts

```diff
@@ -153,8 +150,6 @@ test(testCase, async (t: Test) => {
     );
 
     const { callOutput: helloMsg } = await connector.getContractInfo({
-      contractName,
```

Comment:
@elenaizaguirre I was wondering while looking through this diff: Do we even need the contractAbi and bytecode arguments at this point or can they both be removed in favor of `contractJSON`?

Reply:
You're right. I removed them in favor of contractJSON and keychainPlugin (which saves a contractJSON object too)